x-pack/plugins/synthetics/server/alert_rules/status_rule/monitor_status_rule.ts

```diff
@@ -98,10 +100,20 @@ export const registerSyntheticsStatusCheckRule = (
           fields: getMonitorAlertDocument(monitorSummary),
         });
         const alertUuid = getAlertUuid(alertId);
-        const indexedStartedAt = getAlertStartedDate(alertId) ?? startedAt.toISOString();
+        const alertState = alert.getState() as SyntheticsMonitorStatusAlertState;
+        const errorStartedAt: string = alertState.errorStartedAt || ping['@timestamp'];
+
+        const relativeViewInAppUrl = getRelativeViewInAppUrl({
+          configId,
+          errorStartedAt,
+          locationId,
+        });
 
         const context = {
           ...monitorSummary,
+          errorStartedAt,
+          linkMessage: getFullViewInAppMessage(basePath, spaceId, relativeViewInAppUrl),
+          [VIEW_IN_APP_URL]: getViewInAppUrl(basePath, spaceId, relativeViewInAppUrl),
         };
 
         alert.replaceState({
```

Comment:
@shahzad31 The values present in `updateState` don't seem to make any context in our use case.

They are based on the start time and trigger time of the original rule execution which is global to all our monitors and useless for users and ourselves in this logic.

Should we get rid of these values, i.e. triggerdAt, firstTriggeredAt, firstCheckedAt, checkedAt, etc?